packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -165,4 +165,24 @@ test.describe('Participant pane tests', async () => {
     }, participantStringId);
     expect(await stableScreenshot(page)).toMatchSnapshot('PSTN-participant-pane-callee-name-truncation.png');
   });
+
+  /* @conditional-compile-remove(one-to-n-calling) @conditional-compile-remove(PSTN-calls) */
+  test('Participant shows unknown icon when displayName is missing', async ({ page, serverUrl }, testInfo) => {
```

Comment:
Should we also test the case where the displayName is set by the user serendipitously the same as the localized placeholder?

Reply:
I think that's a side effect.
We want to test the expected behaviour.
If something changes in the future where our logic is more streamlined, the side effect goes away but our test will start failing.